packages/next/src/utils.ts

```diff
@@ -2341,6 +2342,7 @@ export async function getMiddlewareBundle({
                   ...wasmFiles,
                   ...assetFiles,
                 },
+                regions: (edgeFunction.userConfig?.regions ?? 'auto') as any,
```

Comment:
Do you need to update build-utils too?

Reply:
It was already updated I believe? Typecheck passes

Reply:
Why `as any`?

Reply:
Because `unknown` does not match the expected type. Related to the other conversation